packages/Tag/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@paprika/tag",
+  "version": "1.0.0",
+  "description": "The Tag component is for showing metadata like status, type, or levels.",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika.git",
+    "directory": "packages/Tag"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/constants": "^1.0.0",
+    "@paprika/helpers": "^2.0.1",
+    "@paprika/raw-button": "^1.0.3",
+    "@paprika/stylers": "^1.0.0",
+    "@paprika/tokens": "^1.0.0",
+    "@paprika/icon": "^1.0.4",
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "@paprika/l10n": "^1.1.3",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
17? 😳  
I don't think any of our apps meet this.

Reply:
This is what ListboxWithTags component was using, so i went with that.
Ok I'll change to 
```
"react": "^16.8.4",
    "react-dom": "^16.8.4",
    "styled-components": "^4.2.0"
    ```.
   I'll update it in ListboxWithTags component as well

Reply:
🙏 this was actually a copy/paste issue that I forgot to remove, will not break anything but as other have mention in the slack channel we should use the bare minimum version that our component can support :)